src/sentry/runner/initializer.py

```diff
@@ -183,6 +187,8 @@ def bootstrap_options(settings, config=None):
             from .importer import ConfigurationError
 
             raise ConfigurationError("Malformed config.yml file")
+    else:
+        options = {}
```

Comment:
the reordering here helped mypy understand that it's *always* `dict[str, Any]` after this point and that `safe_load` returns `Any` and that it narrows properly to here